src/arch/xtensa/boot_loader.c

```diff
@@ -288,8 +288,8 @@ static int32_t lp_sram_init(void)
 	/* add some delay before writing power registers */
 	idelay(delay_count);
 
-	lspgctl_value = shim_read(SHIM_LSPGCTL);
-	shim_write(SHIM_LSPGCTL, lspgctl_value & !LPSRAM_MASK(0));
+	lspgctl_value = io_reg_read(LSPGISTS);
+	io_reg_write(LSPGCTL, lspgctl_value & !LPSRAM_MASK(0));
```

Comment:
Btw, should this not be ~mask (since it's bitwise) ? If not, we should comment this so other dont think the same as me.